packages/dx-react-scheduler-demos/src/demo-sources/scheduler-all-day-panel/material-ui/week-view.jsx

```diff
@@ -31,12 +59,29 @@ export default class Demo extends React.PureComponent {
           <ViewState
             defaultCurrentDate={currentDate}
           />
+          <EditingState
+            onCommitChanges={this.onCommitChanges}
+          />
           <WeekView
             startDayHour={9}
             endDayHour={19}
           />
+          <WeekView
+            name="Work Week"
+            startDayHour={9}
+            endDayHour={19}
+            excludedDays={[0, 6]}
+          />
+          <MonthView />
           <Appointments />
           <AllDayPanel />
+
+          <Toolbar />
+          <ViewSwitcher />
+
+          <DragDropProvider
+            allowDrag={({ title }) => title !== 'DRAGGING DISABLED'}
```

Comment:
The function is recreated on every `render` though it is actually the same.

Reply:
Okay, I see. This test example for debugging. I remove these code after review.

There are two demos in production - `editing/drag-drop` and `featured-editing`